app/controllers/api/v0/articles_controller.rb

```diff
@@ -21,11 +18,13 @@ class ArticlesController < ApiController
       def index
         @articles = ArticleApiIndexService.new(params).get
 
-        set_surrogate_key_header "articles", @articles.map(&:record_key)
+        set_surrogate_key_header Article.table_key, @articles.map(&:record_key)
```

Comment:
`Article.table_key` is effectively equal to `articles` but I'm following `fastly-rails` conventions to make it explicit :)